product-backend/scripts/populateDataBase.sh

```diff
@@ -1 +1,2 @@
-mysql -h"$DB_HOST" -D"$DB_NAME" -u"$DB_USERNAME" -p"$DB_PASSWORD" < "populate-database-job-roles.sql"
\ No newline at end of file
+mysql -h"$DB_HOST" -D"$DB_NAME" -u"$DB_USERNAME" -p"$DB_PASSWORD" < "populate-database-job-roles.sql"
+mysql -h"$DB_HOST" -D"$DB_NAME" -u"$DB_USERNAME" -p"$DB_PASSWORD" < "populate-database-job-bands.sql"
```

Comment:
Job roles depends on bands
```suggestion
mysql -h"$DB_HOST" -D"$DB_NAME" -u"$DB_USERNAME" -p"$DB_PASSWORD" < "populate-database-job-bands.sql"
mysql -h"$DB_HOST" -D"$DB_NAME" -u"$DB_USERNAME" -p"$DB_PASSWORD" < "populate-database-job-roles.sql"
```